packages/bento-web/src/profile/ProfileDetailPage/components/AssetSection.tsx

```diff
@@ -69,3 +78,11 @@ const Description = styled.p`
   letter-spacing: -0.5px;
   color: rgba(255, 255, 255, 0.8);
 `;
+
+const AssetSkeleton = styled(Skeleton)`
+  width: 536px;
```

Comment:
이거 모바일에선 깨지지 않을까요? `100%` 어때요
```suggestion
  width: 100%;
```